spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -439,8 +437,7 @@ def angle_correction(self):
                 [x_centerline_deriv[iz] * px, y_centerline_deriv[iz] * py, pz]))
 
             # compute the angle between the normal vector of the plane and the vector z
-            angle = np.arccos(np.vdot(tangent_vect, np.array([0, 0, 1])))
-            self.angles[iz] = math.degrees(angle)
+            self.angles[iz] = np.arccos(np.vdot(tangent_vect, np.array([0, 0, 1])))
```

Comment:
Just double-checked, and your change makes sense to me. Since `self.angles` is eventually consumed by `np.cos`:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/6f388046fa7d21d4315513eba8a2ec0b94267adf/spinalcordtoolbox/scripts/sct_analyze_lesion.py#L251

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/6f388046fa7d21d4315513eba8a2ec0b94267adf/spinalcordtoolbox/scripts/sct_analyze_lesion.py#L256

With your change, we're basically going from:

* Before: `np.arccos` (rad) -> `math.degrees` (deg) -> `np.cos` [(expects rad)](https://numpy.org/doc/stable/reference/generated/numpy.cos.html)
* After: `np.arccos` (rad) -> `np.cos` (expects rad)